R-Interface/CMakeLists.txt

```diff
@@ -28,10 +28,8 @@
 
 list(APPEND CMAKE_MESSAGE_CONTEXT R-Interface)
 
-file(GLOB SOURCE_FILES "${CMAKE_CURRENT_LIST_DIR}/jaspResults/src/*.cpp")
 list(APPEND SOURCE_FILES "${CMAKE_CURRENT_LIST_DIR}/jasprcpp.cpp")
```

Comment:
But the APPEND here should be GLOB just like at line 33

Reply:
isn't it `list(APPEND ...)` and `FILE(GLOB ...)`? Cmake throws an error if I do `list(GLOB ... )`

Reply:
I hadn't noticed that it was also a different function, but I interpreted "APPEND" as needing a list to exist already.
However if it works it works